README.md

```diff
@@ -129,6 +138,21 @@ your selected strategy. The strategies available are as follows:
 3) `:null` is a noop. If you want to disable notifications temporarily, you can
   configure the strategy to `:null`.
 
+4) To create your own custom notifier, define a callable class, then configure
+  `JunkDrawer::Notifier` with that class as the strategy.
```

Comment:
Per my comment below, I think any callable should be fine. It might be good to have a couple of examples:

```rb
JunkDrawer::Notifier.strategy = ->(*args) {
  MonitoringServiceA.notify(*args)
  MonitoringServiceB.notify(*args)
}
```